internal/promotion/runner/builtin/git_cloner.go

```diff
@@ -103,10 +103,22 @@ func (g *gitCloner) run(
 			SSHPrivateKey: creds.SSHPrivateKey,
 		}
 	}
+	var repoUser git.User
+
+	if cfg.Author != nil {
+		repoUser = git.User{
+			Name:  cfg.Author.Name,
+			Email: cfg.Author.Email,
+			SigningKey:     cfg.Author.SigningKey, // Optional, may be empty
+		}
+	} else {
+		repoUser = g.gitUser // Default to the system-level gitUser
+	}
```

Comment:
Readability...

The declaration of the `repoUser` variable is closely associated with the lines that follow and not really associated at all with the lines that precede it. If you're going to use a line break anywhere around here, it ought to be placed such that this varaible declaration is, visually, more clearly associated with the lines that follow:

```suggestion

	var repoUser git.User
	if cfg.Author != nil {
		repoUser = git.User{
			Name:  cfg.Author.Name,
			Email: cfg.Author.Email,
			SigningKey:     cfg.Author.SigningKey, // Optional, may be empty
		}
	} else {
		repoUser = g.gitUser // Default to the system-level gitUser
	}
```